sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1214,34 +1213,35 @@ def _from_generated(cls, generated):
         return None
 
 
-class DelimitedTextConfiguration(GenDelimitedTextConfiguration):
-    """Defines the input or output delimited (CSV) serialization for a blob quick query request.
+class JSONEncoder(object):
```

Comment:
this is not necessary probably? service will apply the default one if user don't provide anything

Reply:
Previously there was a JsonnTextConfiguration class, we removed that class because that class only has one attribute: record separator. So I guess we don't want to add JSONEncoder back?
Any thought @annatisch @kasobol-msft

Reply:
This is true - the main reason I think we should add it back is for future compatibility. If - in future - we add a new kind of formatting that also only has one property - we will be backed into a corner and have to add the object anyway.
If we include this object now - and later decide to add a "short cut" by allowing users to only specify the separator character - we can also support that in a non-breaking way.

Reply:
the current name is JSONEncoder, if in the future there's new formatting, it will not share the same name with Json formatting I guess.
In STG71, we changed Encryption Scope form a class to a string because it has only one attribute encryption_scope, so probably here we want to stick to the same thing?

Reply:
The current name is actually `DelimitedJSON` - I updated it since the initial comment.

Reply:
I still feel this class is not necessary, since there's only one attribute inside the class, previously we didn't wrap the single attribute with a class so I guess we don't want to wrap the attribute with a class here either. Any thoughts @kasobol-msft ?

Reply:
Hi Anna 😀 @annatisch you also pointed out we shouldn't force users to construct a class with only one property https://github.com/Azure/azure-sdk-for-python/pull/10946#discussion_r413918843 is there a reason we want to add the class back?

Reply:
I agree with @annatisch we should be open for extension. I can imagine more "formatting knobs" coming in at some point.